.github/workflows/release-labs.yml

```diff
@@ -0,0 +1,49 @@
+name: Release labs Workflow
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run && github.event.workflow_run.conclusion == 'success' || true }}
```

Comment:
We don't need this check as this workflow does not depend on any other.

Reply:
Good catch, fixed.